charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -0,0 +1,49 @@
+- apiVersion: "v1"
+  kindsRegexp: "^namespaces$"
+  resourceNameRegexp: "^fleet-|^cluster-fleet-"
```

Comment:
@nickgerace IMO, it might make sense to start each file with something like:
```yaml
{{- $resourceNameRegexp := "^fleet-|^cluster-fleet-" }}
{{- $resourceNamespaceRegexp := "^fleet-|^cluster-fleet-" }}
```

And then refer to them as:
```yaml
resourceNameRegexp: {{ $resourceNameRegexp }}
namespaceRegexp: {{ $resourceNamespaceRegexp }}
```

That way specific overrides to the regexp expressions can be called out in the template itself, such as adding something like `gitjob` for only certain components.

Reply:
Based on the current version of the commits, I think it's fine to omit this. But this is something probably to consider in the future.